templates/dashboard/includes/_filters.html

```diff
@@ -0,0 +1,45 @@
+{% load i18n %}
+{% load materializecss %}
+
+
+<div class="card">
+  <form method="get">
+    <div class="card-content">
+      <div class="col s12">
+        <span class="card-title">{% trans 'Filters' context 'Dashboard card title' %}</span>
+        <span>{% trans 'Total records found' context 'Dashboard filter' %}: {{ filter.qs|length }}</span>
+      </div>
+      <div class="clearfix"></div>
+    </div>
+    <div class="card-content card-action">
+      <div class="row">
+        {% for field in filter.form %}
+          <div class="col s12">
+            {% if field.name == 'sort_by' %}
```

Comment:
Not a fan of, I think rendering those fields separately would be a more error-proof option.

Reply:
Those fields are taken from many filters, adding every possible field would create enormous if statement.